### DISCUSSION TITLE
Controlling the width of the mesh edges

### DISCUSSION THREAD

**Discussion by fbeaudo** [pyvista/pyvista#2389] [Q&A] [answered]
Suppose that I have an UnstructuredGrid object called grid.

Is it possible to plot the grid edges with finer line width ?

Something like:
`grid.plot(show_edges=True,line_width=0.25)`

I have seen that make line_width=10 results in very thick edges. However, it seems like setting line_width < 1 does not result in finer lines. Is there a minimum linewidth in PyVista or VTK. I would really like to make the edges finer as I have a very fine mesh for which it becomes impossible to distinguish elements with the default edge line width.

**Comment by akaszynski** [comment, marked as answer]
It seems that there's a limitation within vtk to the line width. You can, however, workaround this with:

```py
import pyvista as pv

mesh = pv.Sphere()
edges = mesh.extract_all_edges()

pl = pv.Plotter()
pl.add_mesh(mesh)
act = pl.add_mesh(edges, line_width=0.1, color='k')
pl.show()
```
![tmp](https://user-images.githubusercontent.com/11981631/160829173-3a59a12f-55c7-45ef-bbb4-aa1963f194cc.png)


This will result in thinner lines (down to one pixel wide).
